src/app/clusters/barrier-control-server/barrier-control-server.c

```diff
@@ -41,8 +41,14 @@
 #include "barrier-control-server.h"
 #include "af.h"
 
+#include <assert.h>
+
+#ifndef MILLISECOND_TICKS_PER_DECISECOND
+#define MILLISECOND_TICKS_PER_DECISECOND 1000
```

Comment:
@bzbarsky-apple had a better plan for this than redefinition in every source file